packages/server-core/src/projects/project-commits/project-commits.test.ts

```diff
@@ -69,12 +72,15 @@ describe('project-commits.test', () => {
 
     testUserApiKey = await app.service(userApiKeyPath).create({ userId: testUser.id })
 
-    await app.service(identityProviderPath).create(
+    await app.service(identityProviderPath)._create(
```

Comment:
You can use `identityProviderDataResolver.resolve()` method here to populate default values. Same for other instances